docs/testrunner-toolkit.md

```diff
@@ -33,7 +34,7 @@ JavaScript framework of your choice.
 
 ## Supported Frameworks and Browsers
 
-In the current beta, the toolkit supports the following frameworks: [Cypress](https://github.com/cypress-io/cypress), [Playwright](https://github.com/microsoft/playwright), and [TestCafe](https://github.com/DevExpress/testcafe). The specific framework you use for testing depends on the types of tests you
+In the current beta, the toolkit supports the following frameworks: [Cypress](https://github.com/cypress-io/cypress), [Playwright](https://github.com/microsoft/playwright), [TestCafe](https://github.com/DevExpress/testcafe), and [Espresso](https://developer.android.com/training/testing/espresso). The specific framework you use for testing depends on the types of tests you
 need to run, and the environment where you run the tests.
 
 Furthermore, the framework and browser version support depends on the Sauce Labs docker images. The table below indicates framework and browser support based on the requisite docker image tag.
```

Comment:
Do we need to add an Espresso tab to each of the tables to show the supported OS/browser combinations?

Reply:
The supported platform is only Android and there are no browser combinations, only supported Android versions.